suite/tests/common/nativeexec_exenative.expect

```diff
@@ -0,0 +1,17 @@
+Not under DR!
+calling via IAT-style call
+nativeexec.dll:import_me1(57)
+calling via PLT-style call
+nativeexec.dll:import_me2(37)
+calling via funky ind call
+nativeexec.dll:import_me3(17)
+calling nested native
+nativeexec.exe:print_int(42)
+calling cross-module unwinder
+before longjmp, 1
+after longjmp
+calling indirect ret_imm
+ -> 40
+calling loop_test
+Not under DR!
+all done
```

Comment:
From the test, how do we know we regain control at some where? somewhere did not print "Not under DR"
It would better if we can print "Under DR" somewhere to show we regain the control.